README.md

```diff
@@ -285,6 +286,7 @@ Committee 3.* has many breaking changes so we recommend upgrading to the latest
 2. Run your test suite and fix any deprecation warnings that appear.
 3. Update to the latest 3.* release.
 4. Switch to OpenAPI 3 if you'd like to do so.
+   (If you use GET request body data, don't forget to set allow_get_body true.)
```

Comment:
Let's maybe start a new section after "Test assertions" called "Other changes" and make it a list like:

```
### Other changes

* `GET` request bodies are ignored in OpenAPI 3 by default. If you want to use them, set the `allow_get_body` option to `true`.
```